front/app/utils/streams/index.ts

```diff
@@ -22,46 +24,59 @@ import {
   forEach,
   union,
   uniq,
-  isUndefined,
 } from 'lodash-es';
 import request from 'utils/request';
+import { reportError } from 'utils/loggingUtils';
+import {
+  deepFreeze,
+  sanitizeQueryParameters,
+  removeTrailingSlash,
+  isSingleItemStream,
+  getStreamId,
+} from './utils';
+
+// typings
+import { Observer, Observable, Subscription } from 'rxjs';
+
+// constants
+import { API_PATH } from 'containers/App/constants';
 import { authApiEndpoint } from 'services/auth';
 import { currentAppConfigurationEndpoint } from 'services/appConfiguration';
 import { currentOnboardingCampaignsApiEndpoint } from 'services/onboardingCampaigns';
-import stringify from 'json-stable-stringify';
-import { reportError } from 'utils/loggingUtils';
-import { isUUID } from 'utils/helperUtils';
-import modules from 'modules';
 
 export type pureFn<T> = (arg: T) => T;
+
 type fetchFn = () => Promise<any>;
-interface IObject {
+
+export interface IObject {
   [key: string]: any;
 }
+
 export type IObserver<T> = Observer<T | pureFn<T> | null>;
+
 export interface IStreamParams {
   bodyData?: IObject | null;
   queryParameters?: IObject | null;
   cacheStream?: boolean;
   skipSanitizationFor?: string[];
 }
-export interface IInputStreamParams extends IStreamParams {
+
```

Comment:
This type in only used for `.get`, and `.get` doesn't actually support sending a body (which is part of the reason why we had to add `.analytics`). So I removed `bodyData` as a possible parameter